topo/node/node.go

```diff
@@ -343,10 +343,13 @@ func (n *Impl) CreateService(ctx context.Context) error {
 		if name == "" {
 			name = fmt.Sprintf("port-%d", k)
 		}
+		if v.Outside != 0 {
+			log.Warningf("Outside should not be set by user. The key is used as the target external port")
+		}
 		sp := corev1.ServicePort{
 			Name:       name,
 			Protocol:   "TCP",
-			Port:       int32(v.Inside),
+			Port:       int32(k),
```

Comment:
so to confirm the new behavior is:

port is key, if v.Outside is specified it overwrites it

targetPort is always v.Inside

should targetPort default to the key as well if v.Inside is unspecified? so the user could easily specify a service that uses the same inside and outside port:

```
services: {
  key: 22
  value: {
    name: "ssh"
  }
}
```